types/tinovi/PM-IO-5-SM/default.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$id": "https://akenza.io/decentlab/iam/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "valve": {
+      "title": "Valve",
+      "type": "boolean",
+      "description": "Valve on = true, Valve off = false"
+    },
+    "leak": {
+      "title": "Leak",
+      "type": "boolean",
+      "description": "Leak detected, either true or false"
+    },
+    "soilConductivity": {
+      "title": "Soil conductivity",
+      "unit": "uS/cm",
+      "type": "number",
+      "description": "Conductivity of the soil"
+    },
+    "temperature": {
+      "title": "Temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "The temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "waterContent": {
+      "title": "Water content",
+      "unit": "%",
+      "type": "number",
+      "description": "Water content in the soil",
+      "minimum": 0,
+      "maximum": 100
+    }
```

Comment:
there are quite a few missing data keys

Reply:
These keys are the only ones for this sensor. I could add the rest but we said we would make different devicetypes for different sensors even if they may have the same decoder.